common/syntaxflow/sfvm/visitor.go

```diff
@@ -16,6 +16,7 @@ import (
 type SyntaxFlowVisitor struct {
```

Comment:
this function should simplify? just embed `schema.SyntaxflowRule`, like `Frame`?

Reply:
implement this modify in pr: #1895